etc/adminhtml/system/adyen_security.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!--
+/**
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2022 Adyen N.V. (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+-->
+<include xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_include.xsd">
+    <group id="adyen_pwa" translate="label" type="text" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
This setting is not related to PWA. I agree with @Morerice, this should go into another existing section.

Reply:
Hello @michaelpaul, thanks for the review. `adyen_pwa` is definitely a typo, I am going to fix it. However, I am currently discussing the suitable position with @aziya-im. Then, this PR will be updated.